app/editor/src/features/admin/work-orders/styled/WorkOrderList.tsx

```diff
@@ -22,4 +22,8 @@ export const WorkOrderList = styled(FormPage)`
       color: ${(props) => props.theme.css.primaryLightColor};
     }
   }
+
+  div.column {
+    overflow: hidden;
```

Comment:
Possibly we would want this as the default behavior of all columns, but I don't know.